docs/user-guide/textcleaning.rst

```diff
@@ -0,0 +1,95 @@
+.. _data-curator-text-cleaning:
+
+=========================
+Text Cleaning
+=========================
+
+--------------------
+Overview
+--------------------
+Documents in datasets may contain improperly decoded characters (e.g. "The Mona Lisa doesn't have eyebrows." decoding as "The Mona Lisa doesnÃƒÂ¢Ã¢â€šÂ¬Ã¢â€žÂ¢t have eyebrows."), inconsistent line spacing, and many urls.
+NeMo Curator provides a few modules that can help remove undesirable text from within individual documents.
+
+--------------------
+Use Cases
+--------------------
+* Fixing improperly decoded unicode characters from webpages.
+* Standardizing document layout by removing excessive newlines.
+* Removing URLs in documents.
+
+--------------------
+Modules
+--------------------
+NeMo Curator provides a collection of easy to use modules for cleaning text.
+
+.. code-block:: python
+
+    from nemo_curator import Sequential, Modify, get_client
+    from nemo_curator.datasets import DocumentDataset
+    from nemo_curator.modifiers import UnicodeReformatter, UrlRemover, NewlineNormalizer
+
+    def main():
+        client = get_client(cluster_type="cpu")
+
+        dataset = DocumentDataset.read_json("books.jsonl")
+        cleaning_pipeline = Sequential([
+            Modify(UnicodeReformatter()),
+            Modify(NewlineNormalizer()),
+            Modify(UrlRemover()),
+        ])
+
+        cleaned_dataset = cleaning_pipeline(dataset)
+
+        cleaned_dataset.to_json("cleaned_books.jsonl")
+
+    if __name__ == "__main__":
+        main()
+
+Here, we load a dataset and perform all of the cleaning operations that NeMo Curator supports.
+* ``Modify(UnicodeReformatter())``: Uses `ftfy <https://ftfy.readthedocs.io/en/latest/>`_ to fix broken Unicode characters. Modifies the `"text"` field of the datset by default. This can be changed by setting ``Modify(UnicodeReformatter(), text_field="my_field")``.
```

Comment:
```suggestion
* ``Modify(UnicodeReformatter())``: Uses `ftfy <https://ftfy.readthedocs.io/en/latest/>`_ to fix broken Unicode characters. Modifies the `"text"` field of the dataset by default. This can be changed by setting ``Modify(UnicodeReformatter(), text_field="my_field")``.
```